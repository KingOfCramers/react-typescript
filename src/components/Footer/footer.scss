.footer {
  color: red;
}

$theme-colors: (
  "primary": "blue",
);
