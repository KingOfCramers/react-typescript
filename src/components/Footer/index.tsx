import React from "react";

interface FooterProps {}

const Footer = (props: FooterProps) => {
  return (
    <div className={"footer"}>
      <div>Hi</div> Copyright 2020, Harrison Cramer
    </div>
  );
};

export default Footer;
