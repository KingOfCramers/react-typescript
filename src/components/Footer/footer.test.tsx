import React from "react";
import { shallow } from "enzyme";
import Footer from "./index";

test("renders the component", () => {
  const component = shallow(<Footer />);
  expect(component).toMatchSnapshot();
});
