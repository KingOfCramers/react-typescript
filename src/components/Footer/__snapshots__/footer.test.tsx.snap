// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<div
  className="footer"
>
  Footer
</div>
`;
